src/box/iproto_constants.c

```diff
@@ -317,3 +317,44 @@ const char *vy_row_index_key_strs[VY_ROW_INDEX_KEY_MAX] = {
 	NULL,
 	"row index",
 };
+
+#define IPROTO_RECORD_MEMBER(s, v) {.name = #s, .value = v},
```

Comment:
Ditto.

```suggestion
#define IPROTO_CONSTANT_MEMBER(s, v) {.name = #s, .value = v},
```

Reply:
Maybe we could an `iproto_util` header for this?

Reply:
It's a trivial macro, so I think it's OK to implement it twice.

Reply:
Renamed macro